GitTfs/Commands/Fetch.cs

```diff
@@ -52,6 +55,18 @@ public string BatchSizeOption
             }
         }
 
+        int upToChangeSet { get; set; }
+        public string UpToChangeSetOption
+        {
+            set
+            {
+                int tmp;
```

Comment:
could you rename this variable to something more explicit, like `changesetIdParsed` for example?